CMakeLists.txt

```diff
@@ -8,44 +8,24 @@ list(APPEND SOURCES "lvgl_tft/disp_driver.c")
 
 # Include only the source file of the selected
 # display controller.
-if(CONFIG_LV_TFT_DISPLAY_CONTROLLER_ILI9341)
```

Comment:
Thanks for fixing this, it was introduced by me when all the drivers had the same API names, so quite a while ago. This also should make using multiple displays in the same project, isn't?

Reply:
Yes, compiling all the drivers is prerequisite for multiple displays setup.

But more must be done: currently the (single) display is configured via Kconfig, which is reasonably complex for one display. For multiple displays we would have to dynamically add configuration menus for each enabled display, and then assign the Kconfig options to the correct driver, which is something I wouldn't recommend for number of reasons:
- lower maintainability
- higher complexity
- community knowledge of Kconfig, etc.

That's why I am always for runtime configuration.
(We could still use Kconfig for the naive but frequent usecase of connecting one display with default configuration to the MCU, but we can talk about it later)

Reply:
Sure, I agree that having a runtime configuration is a way to go for complex projects.